source/_components/binary_sensor.ring.markdown

```diff
@@ -0,0 +1,38 @@
+---
+layout: page
+title: "Ring"
+description: "Instructions on how to integrate your Ring.com devices within Home Assistant."
+date: 2017-03-10 10:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: ring.png
+ha_category: Binary Sensor
+ha_release: 0.40
+---
+
+The `ring` binary sensor allows you to integrate your [Ring.com](https://ring.com/) devices in Home Assistant.
+
+Currently only doorbells are supported by this sensor.
+
+To enable device linked in your [Ring.com](https://ring.com/) account, add the following to your `configuration.yaml` file:
+
+```yaml
+# Example configuration.yaml entry
+binary_sensor:
+  - platform: ring
+    username: USERNAME
+    password: PASSWORD
+    monitored_conditions:
+      - ding
+      - motion
+```
+
+Configuration variables:
+
+- **username** (*Required*): The username for accessing your Ring account.
+- **password** (*Required*): The password for accessing your Ring account.
+- **monitored_conditions** array (*Required*): Conditions to display in the frontend. The following conditions can be monitored.
```

Comment:
Hello @Landrash, 

By default there are not included or monitored. The user must specify which sensor to use. 

https://github.com/tchellomello/home-assistant/blob/binary_sensor_ring/homeassistant/components/binary_sensor/ring.py#L46

The user can choose only one or 2 sensors. They are not dependent.